source/tests/system/nirfmxinstr_driver_api_tests.cpp

```diff
@@ -370,6 +370,15 @@ TEST_F(NiRFmxInstrDriverApiTests, TimestampFromValuesRoundTrip_SucceedsWithOrigi
   EXPECT_NEAR(FRACTIONAL_SECONDS, values_response.fractional_seconds(), .001);
 }
 
+TEST_F(NiRFmxInstrDriverApiTests, BuildPortString_SucceedsWithExpectedValue)
+{
+  constexpr auto EXPECTED_PORT_STRING = "port::RFSA1/0";
+  const auto build_port_string_response = client::build_port_string(stub(), "", "", "RFSA1", 0);
```

Comment:
Should this use a 2-digit channel number?

Reply:
Yes, changed it to pass in 10 for the channel number and expect "port::RFSA1/10".